docs/source/installation.md

```diff
@@ -128,11 +128,35 @@ cd "${SRCROOT}/${TARGET_NAME}"
 "${SCRIPT_PATH}"/check-and-run-apollo-cli.sh codegen:generate --target=swift --includes=./**/*.graphql --localSchemaFile="schema.json" API.swift
 ```
 
+### If you are integrating Apollo using SPM + Xcode 11 [BETA]
+
+> NOTE: These instructions are as of Xcode 11, beta 4. Please file an issue if anything has changed in newer betas or in the final release!
+
+If you're using Xcode 11, SPM will check out the appropriate build script along with the rest of the files. Add the following to your run script to find the script in the appropriate place: 
+
+```sh
+# Go to the build root and go back up to where SPM keeps the apollo iOS framework checked out.
+cd "${BUILD_ROOT}"
+cd "../../SourcePackages/checkouts/apollo-ios/scripts"
```

Comment:
If anyone's got a better build var to use for this that would at least get us directly to `SourcePackages`, let me know! I went cross-eyed staring at environment variables but I couldn't find anything that wasn't under `Build/Products`